crates/editor/src/editor.rs

```diff
@@ -7414,6 +7414,12 @@ impl Editor {
             return;
         };
 
+        // Only report telemetry events for online providers managed by Zed's service,
+        // not for third-party providers like Ollama that don't count toward online usage
+        if provider.name() != "zed-predict" {
```

Comment:
I don't think we want to do this. We collect telemetry for all providers for informational purposes (so we can see which ones are most used).

The usage counting/limiting for Zed's Edit Predictions is handled separately from telemetry.

Reply:
@maxdeviant,
I suppose that what you offer will lead to the fact that more people will refuse telemetry, at least. The next logical step is to add a list of everything found in `$PATH` to telemetry?

Reply:
Thanks, makes sense - I wasn't aware the counter was a separate thing. Reverted